nexus/test-utils/src/resource_helpers.rs

```diff
@@ -30,46 +32,83 @@ where
         .unwrap()
 }
 
-pub async fn create_organization(
+pub async fn object_create<InputType, OutputType>(
     client: &ClientTestContext,
-    organization_name: &str,
-) -> Organization {
-    let input = params::OrganizationCreate {
-        identity: IdentityMetadataCreateParams {
-            name: organization_name.parse().unwrap(),
-            description: "an org".to_string(),
-        },
-    };
-    NexusRequest::objects_post(client, "/organizations", &input)
+    path: &str,
+    input: &InputType,
+) -> OutputType
+where
+    InputType: serde::Serialize,
+    OutputType: serde::de::DeserializeOwned,
+{
+    NexusRequest::objects_post(client, path, input)
         .authn_as(AuthnMode::PrivilegedUser)
         .execute()
         .await
-        .expect("failed to make request")
+        .expect("failed to make \"create\" request")
         .parsed_body()
         .unwrap()
 }
 
+pub async fn create_organization(
+    client: &ClientTestContext,
+    organization_name: &str,
+) -> Organization {
+    object_create(
+        client,
+        "/organizations",
+        &params::OrganizationCreate {
+            identity: IdentityMetadataCreateParams {
+                name: organization_name.parse().unwrap(),
+                description: "an org".to_string(),
+            },
+        },
+    )
+    .await
+}
+
 pub async fn create_project(
     client: &ClientTestContext,
     organization_name: &str,
     project_name: &str,
 ) -> Project {
-    NexusRequest::objects_post(
+    let url = format!("/organizations/{}/projects", &organization_name);
+    object_create(
         client,
-        &format!("/organizations/{}/projects", &organization_name),
+        &url,
         &params::ProjectCreate {
             identity: IdentityMetadataCreateParams {
                 name: project_name.parse().unwrap(),
                 description: "a pier".to_string(),
             },
         },
     )
-    .authn_as(AuthnMode::PrivilegedUser)
-    .execute()
     .await
-    .expect("failed to make request")
-    .parsed_body()
-    .unwrap()
+}
+
+pub async fn create_disk(
+    client: &ClientTestContext,
+    organization_name: &str,
+    project_name: &str,
+    disk_name: &str,
+) -> Disk {
+    let url = format!(
+        "/organizations/{}/projects/{}/disks",
+        organization_name, project_name
+    );
+    object_create(
+        client,
+        &url,
+        &params::DiskCreate {
+            identity: IdentityMetadataCreateParams {
+                name: disk_name.parse().unwrap(),
+                description: String::from("sells rainsticks"),
+            },
+            snapshot_id: None,
+            size: ByteCount::from_gibibytes_u32(1),
```

Comment:
Agreed.  It occurred to me while adding this that the analogs for Organization and Project don't take create-time arguments, but that's because there basically aren't any that are useful for testing.  If (or when) we find it's useful for Disks it should be easy to add.